content/pt-br/docs/reference/scheduling-policies/policies.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política de escalonador rodando os seguintes comandos:
+```
+ `kube-scheduler --policy-config-file <filename>` 
+ ou
+  `kube-scheduler --policy-configmap <ConfigMap>`.
+ ```
+
+Essa política escalonadora não é compatível desde a v1.23 do Kubernetes. Flags associadas `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` e `use-legacy-policy-config` também não são compatíveis. Em vez disso, veja [configuração do escalonador](/docs/reference/scheduling/config/) para conseguir um comportamento similar.
+
+## {{% heading "whatsnext" %}}
+
+* Descubra mais sobre o [escalonamento](/docs/concepts/scheduling-eviction/kube-scheduler/)
```

Comment:
This page is translated, so this link can be updated. I would also write "escalonador" in lieu of "escalonamento".